website/server/controllers/api-v3/members.js

```diff
@@ -82,9 +83,34 @@ api.getMember = {
  * @apiSuccess {Number} data.*.achievements.index The unique index assigned to the achievement (only for sorting purposes)
  * @apiSuccess {Anything} data.*.achievements.value The value related to the achievement (if applicable)
  * @apiSuccess {Number} data.*.achievements.optionalCount The count related to the achievement (if applicable)
+
+ * @apiSuccessExample {json} Successful Response
```

Comment:
I kinda forgot to add "basic", "seasonal" and "special" to the example 😅 , what about now?